client/src/state/campaign/CampaignActions.js

```diff
@@ -65,3 +65,16 @@ export function publishCampaign(publishApi, recordType, campaignId) {
       });
   };
 }
+
+/**
+ * Set new campaign created flag
+ *
+ * @param {boolean}
+ * @return {Object}
+ */
+export function setNewCampaignCreated(newItemCreated) {
```

Comment:
naming is inconsistent with what it's doing, maybe `setIsNewItem` or `markHasCreatedItem`

having both `New` and `Created` is double up on meaning.